lte/gateway/c/core/oai/tasks/nas5g/src/ies/M5GSMobileIdentity.cpp

```diff
@@ -194,6 +194,35 @@ int M5GSMobileIdentityMsg::DecodeImsiMobileIdentityMsg(
 
   MLOG(MDEBUG) << "Length :  " << int(ielen);
   memcpy(&imsi->scheme_output, buffer + decoded, ielen - decoded);
+
+  if ((imsi->type_of_identity == M5GSMobileIdentityMsg_SUCI_IMSI) &&
+      (imsi->protect_schm_id != 0)) {
+    MLOG(MDEBUG) << "  SUCI Registration is enabled with protect_schm_id= "
+                 << std::hex << int(imsi->protect_schm_id);
```

Comment:
Please use OAILOG everywhere in this file. MLOG should be sparingly used (e.g., in GRPC clients). Note that I am asking this change throughout this file and not only for the new changes.

Reply:
All the LOGS implemented in nas5g is directed to syslog using MLOG framework, not uses OAILOG, so utilized the same.

Reply:
we should not mix and match. Currently, it is all over the place. Either we land this and you prep a separate PR that changes all the MLOGs to OAILOG or everytime a file is touched, its log messages are updated piece by piece. @ssanadhya, @ardzoht  what is your opinion on this?

Reply:
Agreed, we should use OAILOGs as much as possible. @Kaleem-Wavelabs, could you please open a separate issue to track this? It will be good to fix this soon, so that we can set the precedent for all incoming changes.

Reply:
Agree fully on this, OAILOG should be used for all core transactions logging, MLOG should be reserved for reporting external facing transactions (gRPC calls, event logging, etc.)

Reply:
Raised https://github.com/magma/magma/issues/10650 to address this issue. We will prioritize it to set the platform for the incoming changes. Thanks.